lib/core/src/display/scene.rs

```diff
@@ -181,13 +182,13 @@ impl Mouse {
         let button2_pressed_ref = button2_pressed.clone_ref();
         let button3_pressed_ref = button3_pressed.clone_ref();
         let button4_pressed_ref = button4_pressed.clone_ref();
-        let on_down_handle      = mouse_manager.on_down.add(move |event:&mouse2::event::OnDown| {
+        let on_down_handle      = mouse_manager.on_down.add(move |event:&mouse::event::OnDown| {
             match event.button() {
-                mouse2::Button0 => button0_pressed_ref.set(true),
-                mouse2::Button1 => button1_pressed_ref.set(true),
-                mouse2::Button2 => button2_pressed_ref.set(true),
-                mouse2::Button3 => button3_pressed_ref.set(true),
-                mouse2::Button4 => button4_pressed_ref.set(true),
+                Button0 => button0_pressed_ref.set(true),
+                Button1 => button1_pressed_ref.set(true),
+                Button2 => button2_pressed_ref.set(true),
+                Button3 => button3_pressed_ref.set(true),
+                Button4 => button4_pressed_ref.set(true),
             }
```

Comment:
I would really like to keep here `mouse::ButtonX` instead. I do not think that importing all button names to scope is a good idea as it pollutes the scope and also makes the code harder to read. Seeing "mouse::Button" in code makes it much more obvious.